### DISCUSSION TITLE
Is it possible to translate?

### DISCUSSION THREAD

**Discussion by ivancostajr** [domferr/tilingshell#79] [General]
Hey, guys!

I loved this tiling extension for Gnome and I want to help with a translation into Portuguese. Is it possible?

**Comment by domferr** [comment]
Hey! I was thinking about supporting translation, your help would be awesome! I may start working on the required source code and then you will be ready to translate in Portuguese. Do you have already any experience on translating GNOME extensions? If you haven't, don't worry! I'm just curious

**Reply by ivancostajr** [reply]
I've never translated an extension for Gnome, but I believe it's similar to other programs and apps, using Poedit or similar.

**Reply by domferr** [reply]
Hey, I finally got time to set up everything needed by Tiling Shell to support translations. It will be shipped with the next upcoming version (v14.0). I can confirm it is similar, using Poedit for example.

To let you start translating to Portuguese, what files do you need? Is [this](https://github.com/domferr/tilingshell/blob/13.2/translations/tilingshell%40ferrarodomenico.com.pot) the only one? Thank you!
